addon/package.json

```diff
@@ -86,9 +63,37 @@
     "access": "public",
     "registry": "https://registry.npmjs.org"
   },
+  "files": [
+    "declarations",
+    "dist",
+    "src",
+    "addon-main.cjs"
+  ],
+  "exports": {
+    ".": {
+      "types": "./declarations/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./__private__/types": {
```

Comment:
used by tests only. there were no import restrictions in the prior version of test-waiters.